core-bundle/tests/String/SimpleTokenParserTest.php

```diff
@@ -644,10 +644,6 @@ public function getFunctions(): array
 
     private function getParser(SimpleTokenExpressionLanguage $expressionLanguage = null): SimpleTokenParser
     {
-        if (null === $expressionLanguage) {
-            $expressionLanguage = new SimpleTokenExpressionLanguage();
-        }
-
-        return new SimpleTokenParser($expressionLanguage);
+        return new SimpleTokenParser($expressionLanguage ?? new SimpleTokenExpressionLanguage());
```

Comment:
Same codestyle here:

```suggestion
	$expressionLanguage ??= new SimpleTokenExpressionLanguage();

        return new SimpleTokenParser($expressionLanguage);
```